iron-fit-behavior.html

```diff
@@ -171,17 +278,14 @@
      * the memoized data.
      */
     resetFit: function() {
-      if (!this._fitInfo || !this._fitInfo.sizedBy.width) {
-        this.sizingTarget.style.maxWidth = '';
+      var info = this._fitInfo || {};
+      for (var property in info.sizerInlineStyle) {
```

Comment:
If `this._fitInfo` is ever false-y (which you are explicitly guarding against in the line above this), and you try to do `var property in info.sizerInlineStyle`, this will cause an exception to be thrown. You cannot use `in` with an `undefined` value on the RHS.